macros/parsers/parserGraphTool.pl

```diff
@@ -100,109 +100,158 @@ =head1 GRAPH OBJECTS
+=item snapSizeX, snapSizeY (Default: C<< snapSizeX => 1, snapSizeY => 1 >>)
 
 These restrict the x coordinate and y coordinate of points that can be graphed to being
 multiples of the respective parameter.  These values must be greater than zero.
 
-=item showCoordinateHints (Default: showCoordinateHints => 1)
+=item showCoordinateHints (Default: C<< showCoordinateHints => 1 >>)
 
 Set this to 0 to disable the display of the coordinates in the lower right corner of the graph.
 
-=item availableTools (Default: availableTools => [ "LineTool", "CircleTool",
-    "VerticalParabolaTool", "HorizontalParabolaTool", "FillTool", "SolidDashTool" ])
+=item availableTools (Default: C<< availableTools => [ "LineTool", "CircleTool",
+    "VerticalParabolaTool", "HorizontalParabolaTool", "FillTool", "SolidDashTool" ] >>)
 
 This is an array of tools that will be made available for students to use in the graph tool.
 The order the tools are listed here will also be the order the tools are presented in the graph
 tool button box.  All of the tools that may be included are listed in the default options above,
 except for the "PointTool", the three point "QuadraticTool", and the four point "CubicTool".
 Note that the case of the tool names must match what is shown.
 
-=item staticObjects (Default: staticObjects => [])
+=item staticObjects (Default: C<< staticObjects => [] >>)
 
 This is an array of fixed objects that will be displayed on the graph.  These objects will not
 be able to be moved around.  The format for these objects is the same as those that are passed
 to the GraphTool constructor as the correct answers.
 
-=item printGraph (Default: undefined)
+=item printGraph (Default: C<undef>)
 
 If the JSXGraphOptions option is set directly, then you will also need to provide a function that
 will generate the corresponding hard copy graph.  Otherwise the hard copy graph will still be
-generated using the above options, and will not look the same as the java script graph.
+generated using the above options, and will not look the same as the JavaScript graph.
 
-=item cmpOptions (Default: cmpOptions => {})
+=item cmpOptions (Default: C<< cmpOptions => {} >>)
 
-This is a hash of options that will be passed to the cmp() method.  These options can also be
-passed as parameters directly to the GraphTool object's cmp() method.
+This is a hash of options that will be passed to the C<cmp()> method.  These options can also be
+passed as parameters directly to the GraphTool object's C<cmp()> method.
 
-=item texSize (Default: texSize => 400)
+=item texSize (Default: C<< texSize => 400 >>)
 
 This is the size of the graph that will be output when a hard copy of the problem is generated.
 
+=item numberLine (Default: C<< numberLine => 0 >>)
+
+If set to 1, then the graph will show both the horizontal and vertical axes.  This is the
+default. If set to 0, then only the horizontal axis will be shown, and the graph can be
```

Comment:
Are your descriptions for `1` and `0` reversed here?

Reply:
Yep.  Thanks.